--- /etc/lighttpd/lighttpd.conf.orig    2018-11-04 14:29:00.700077032 +0000
+++ /etc/lighttpd/lighttpd.conf 2018-11-04 14:29:38.568699677 +0000
@@ -22,6 +22,9 @@
 compress.filetype           = ( "application/javascript", "text/css", "text/html", "text/plain" )

 # default listening port for IPv6 falls back to the IPv4 port
-include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
+#include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
 include_shell "/usr/share/lighttpd/create-mime.assign.pl"
 include_shell "/usr/share/lighttpd/include-conf-enabled.pl"
+
+server.bind = "10.1.2.2"
+$SERVER["socket"] == "[2001:678:3fc:e::2]:80" { server.use-ipv6 = "enable" }
