--- lighttpd.conf.orig  2018-10-15 15:42:12.345480008 +0000
+++ lighttpd.conf       2018-10-15 15:43:50.091036311 +0000
@@ -22,6 +22,9 @@
 compress.filetype           = ( "application/javascript", "text/css", "text/html", "text/plain" )

 # default listening port for IPv6 falls back to the IPv4 port
-include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
+#include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
 include_shell "/usr/share/lighttpd/create-mime.assign.pl"
 include_shell "/usr/share/lighttpd/include-conf-enabled.pl"
+
+server.bind = "10.1.3.2"
+$SERVER["socket"] == "[2001:678:3fc:f::2]:80" { server.use-ipv6 = "enable" }
