<![CDATA[
--- squidGuard.conf.dpkg-dist	2008-11-10 11:17:36.000000000 +0100
+++ squidGuard.conf.new	2008-11-10 11:15:29.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 dbhome /var/lib/squidguard/db
-logdir /var/log/squid
+logdir /var/log/squid3
 
 #
 # TIME RULES:
@@ -46,19 +46,106 @@
 # DESTINATION CLASSES:
 #
 
-dest good {
+dest adult {
+	domainlist	adult/domains
+	expressionlist	adult/expressions
+	urllist		adult/urls
 }
 
-dest local {
+dest agressif {
+	domainlist	agressif/domains
+	expressionlist	agressif/expressions
+	urllist		agressif/urls
 }
 
-#dest adult {
-#	domainlist	adult/domains
-#	urllist		adult/urls
-#	expressionlist	adult/expressions
-#	redirect 	http://admin.foo.bar.no/cgi-bin/squidGuard.cgi?clientaddr=%a+clientname=%n+clientident=%i+srcclass=%s+targetclass=%t+url=%u
-#}
+dest astrology {
+	domainlist	astrology/domains
+	urllist		astrology/urls
+}
+
+dest dangerous_material {
+	domainlist	dangerous_material/domains
+	urllist		dangerous_material/urls
+}
+
+dest drogue {
+	domainlist	drogue/domains
+	urllist		drogue/urls
+}
+
+dest financial {
+	domainlist	financial/domains
+	urllist		financial/urls
+}
+
+dest forums {
+	domainlist	forums/domains
+	urllist		forums/urls
+}
+
+dest gambling {
+	domainlist	gambling/domains
+	urllist		gambling/urls
+}
+
+dest games {
+	domainlist	games/domains
+	urllist		games/urls
+}
+
+dest hacking {
+	domainlist	hacking/domains
+	urllist		hacking/urls
+}
+
+dest malware {
+	domainlist	malware/domains
+}
 
+dest marketingware {
+	domainlist	marketingware/domains
+}
+
+dest mixed_adult {
+	domainlist	mixed_adult/domains
+}
+
+dest phishing {
+	domainlist	phishing/domains
+}
+
+dest redirector {
+	domainlist	redirector/domains
+	expressionlist	redirector/expressions
+	urllist		redirector/urls
+}
+
+dest sect {
+	domainlist	sect/domains
+	urllist		sect/urls
+}
+
+dest strict_redirector {
+	domainlist	strict_redirector/domains
+	expressionlist	strict_redirector/expressions
+	urllist		strict_redirector/urls
+}
+
+dest strong_redirector {
+	domainlist	strong_redirector/domains
+	expressionlist	strong_redirector/expressions
+	urllist		strong_redirector/urls
+}
+
+dest warez {
+	domainlist	warez/domains
+	expressionlist	warez/expressions
+	urllist		warez/urls
+}
+
+dest white {
+        domainlist      white/domains
+}
 
 acl {
 #	admin {
@@ -76,8 +163,13 @@
 #	}
 
 	default {
-		pass	 local none
+		pass	white 
+		pass	!adult !agressif !astrology !dangerous_material !drogue
+		pass	!financial !forums !gambling !games !hacking !malware
+		pass	!marketingware !mixed_adult !phishing !redirector !sect
+		pass	!strict_redirector !strong_redirector !warez all
+
 #		rewrite	 dmz
-#		redirect http://admin.foo.bar.no/cgi-bin/squidGuard.cgi?clientaddr=%a+clientname=%n+clientident=%i+srcclass=%s+targetclass=%t+url=%u
+		redirect http://www.stri/cgi-bin/squidGuard.cgi?clientaddr=%a+clientname=%n+clientident=%i+srcclass=%s+targetclass=%t+url=%u
 	}
 }
]]>
