<![CDATA[
--- squidGuard.cgi	2008-11-10 18:37:30.326524640 +0100
+++ /usr/lib/cgi-bin/squidGuard.cgi	2008-11-10 18:35:19.000000000 +0100
@@ -64,10 +64,11 @@
 		"nl (Nederlands),",
 		"no (norsk)."
 	       );
-$image       = "/images/blocked.gif";					# RELATIVE TO DOCUMENT_ROOT
-$redirect    = "http://admin.your-domain/images/blocked.gif";		# "" TO AVOID REDIRECTION
-$proxy       = "proxy.your-domain";					#
-$proxymaster = "operator\@your-domain";				#
+$image       = "/images/access_denied.jpg";				# RELATIVE TO DOCUMENT_ROOT
+$redirect    = "http://www.stri/images/access_denied.jpg";		# "" TO AVOID REDIRECTION
+$proxy       = "proxy.stri";					#
+$proxymaster = "abuse\@stri";					#
+$lang        = "fr";
 $autoinaddr  = 2;			# 0|1|2;
 					# 0 TO NOT REDIRECT
 					# 1 TO AUTORESOLVE & REDIRECT IF UNIQUE
@@ -482,14 +483,14 @@
 	@supported ];
   
   %logo->{"default"}->{"url"}
-    = "http://www.squidguard.org/images/squidGuard.gif";
+    = "http://www.squidguard.org/Logos/squidGuard.gif";
   %logo->{"default"}->{"href"}
     = "http://www.squidguard.org/";
 
   %logo->{"default"}->{"url"}
-    = "http://info.your-domain/images/eto.small.gif";
+    = "http://www.squidguard.org/Logos/squidGuard.gif";
   %logo->{"default"}->{"href"}
-    = "http://www.your-domain/";
+    = "http://www.stri/";
 }
 #
 # END OF CONFIGURABLE OPTIONS
@@ -608,7 +609,10 @@
       print "       $text\n";
     }
   }
-  print "  </TITLE>\n </HEAD>\n";
+  print "  </TITLE>\n";
+  print " <meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\" />\n";
+  print " <meta name=\"Content-Language\" content=\"fr\" />\n";
+  print " </HEAD>\n";
   print " <BODY BGCOLOR=\"#FFFFFF\">\n";
   print "  <TABLE BORDER=0 ALIGN=CENTER WIDTH=100%>\n";
   print "   <TR>\n";
]]>
